playwright/README.md

```diff
@@ -56,36 +59,65 @@ This mode is intended for frontend developers to write their own feature test ca
 For local testing, the following variables need to be loaded into your environment.
 
 ```env
+PW_XFD_URL=http://localhost
 PW_XFD_USERNAME
 PW_XFD_PASSWORD
 PW_XFD_2FA_ISSUER
 PW_XFD_2FA_SECRET
 PW_XFD_USER_ROLE
 PW_XFD_LOGIN
+PW_HEADLESS=false
+PW_CI=false
 ```
 
+In local testing, `PW_HEADLESS` can optionally be set to true or false. `PW_CI` should be set to false, as this variable will adjust the behavior of how the global setup will load the other environment variables.
+
 ### **Environment Variables for Local Testing in VS Code**
 
 If you are using testing in VS Code using the Playwright extension, add the following lines you your `settings.json`, when is accessed from `Extension Settings->Edit in settings.json`
 
 ```json
 "playwright.env": {
-        "PW_XFD_URL":"",
+        "PW_XFD_URL":"http://localhost",
         "PW_XFD_USERNAME":"",
         "PW_XFD_PASSWORD":"",
         "PW_XFD_2FA_ISSUER":"",
         "PW_XFD_2FA_SECRET":"",
         "PW_XFD_USER_ROLE":"",
-        "PW_XFD_LOGIN" : ""
+        "PW_XFD_LOGIN" : "",
+        "PW_HEADLESS" : "false",
+        "PW_CI": "false"
 }
 ```
 
 ## **Local Testing via Docker**
 
+For running in a Dockerzied environment, the following variable should be set in the `xfd/playwright/.env`
+
+```env
+PW_XFD_URL=http://frontend:3000
+PW_XFD_USERNAME
+PW_XFD_PASSWORD
+PW_XFD_2FA_ISSUER
+PW_XFD_2FA_SECRET
+PW_XFD_USER_ROLE
+PW_XFD_LOGIN
+PW_HEADLESS=true
+PW_CI=false
+```
+
 This mode is intended to kick off as part of the build process. A docker container will be created using the same Playwright test.
 
 A wait feature will listen for the frontend to begin accepting requests, as the frontend will take longer to compile than Playwright to be ready for testing. When a code 200 is received from the frontend, the Playwright tests will begin.
 
+For this Dockerized environment, `PW_HEADLESS` must be set to true and `PW_CI` must be set to false.
+
+## **Github Actions Testing with Amazon ECS**
+
+In this mode, the Playwright tests are run everytime changes are committed to the develop and integration branches `xfd/frontend/` directory, or to the `xfd/.github/regression.yml` file itself. The Regression Testing workflow (which so far only encompasses Playwright) calls out to a containerized version of Crossfeed's Playwright testing suite stored on Amazon ECR via an ECS task. When the task is triggered, Playwright ECS will run against either the staging-cd or integration instances of Playwright. Test results are stored in Amazon S3 bucket and downloaded as artifacts to the Github Actions workflow.
```

Comment:
need to fix path - XFD/.github/workflows/regression.yml

Reply:
Path fixed in latest commit: 281c4898